spec/uap_clj/common_spec.clj

```diff
@@ -1,3 +1,27 @@
-(ns uap-clj.common-spec)
+(ns uap-clj.common-spec
+  (:require [clojure.java.io :as io :refer [resource]]
+            [clj-yaml.core :refer [parse-string default-loader-options]])
+  (:import [org.yaml.snakeyaml LoaderOptions]))
 
-(def unknown-ua "Unknown new useragent in the wild/v0.1.0")
+(def ^:const unknown-ua "Unknown new useragent in the wild/v0.1.0")
+
+(defn default-codepoint-limit
+  []
+  (let [options (LoaderOptions.)]
+    (.getCodePointLimit options)))
+
+(defn snakeyaml-loader-options
+  ^LoaderOptions [n]
```

Comment:
Should this `^LoaderOptions` be annotated on the 1st argument rather than the argument vector itself?

Reply:
> Should this `^LoaderOptions` be annotated on the 1st argument rather than the argument vector itself?

This is a type hint on the _return_ value.

Reply:
No, the first argument is an int and `LoaderOptions` is the type of the return value.

Reply:
> No, the first argument is an int and `LoaderOptions` is the type of the return value.

Yep, basically what I replied, with the additional clarification that the arg is an `int`; that.